src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -161,7 +161,6 @@ export class Kernel implements IKernel {
         }
         const promise = this.kernelExecution.executeCell(notebookPromise, cell);
         this.trackNotebookCellPerceivedColdTime(stopWatch, notebookPromise, promise).catch(noop);
-        await promise;
         return promise;
```

Comment:
There's no point awaiting & returning the same promise, came across this code while looking into an nbgolden issue, just sipmle cleanup